WordPress/src/main/res/values/strings.xml

```diff
@@ -1676,6 +1676,10 @@
     <string name="photo_picker_choose_video">Choose video from device</string>
     <string name="photo_picker_capture_photo">Take photo</string>
     <string name="photo_picker_capture_video">Take video</string>
+    <string name="photo_picker_soft_ask_label">%s needs permissions to access your photos</string>
+    <string name="photo_picker_soft_ask_allow">Allow</string>
+    <string name="photo_picker_soft_ask_permissions_denied">%s was denied permissions to access your photos</string>
```

Comment:
As per chat convo, we've agreed that `Photos` is the better choice in this context.